testing/jenkins.py

```diff
@@ -16,18 +17,50 @@
 DCOS_SERVICE_URL = dcos_service_url('jenkins')
 
 
-def create_job(service_name, job_name, timeout_seconds=TIMEOUT_SECONDS):   
+def wait_for_jenkins_marathon_app_healthy(service_name, timeout_seconds=TIMEOUT_SECONDS):
```

Comment:
This function and the one below don't work.  I wrote them and they're not needed.

Reply:
ack